ward/testing.py

```diff
@@ -472,6 +472,14 @@ def display_name(self):
         assert len(display_names) == len(TestOutcome)
         return display_names[self]
 
+    @property
+    def will_fail_session(self) -> bool:
```

Comment:
Could we add tests around the two properties that were added here?

Reply:
Done in https://github.com/darrenburns/ward/pull/233/commits/fc864cc344ca569f1e6bc641c49960ff234eb338